numba_cuda/numba/cuda/dispatcher.py

```diff
@@ -1169,6 +1175,30 @@ def inspect_asm(self, signature=None):
                     for sig, overload in self.overloads.items()
                 }
 
+    def inspect_lto_ptx(self, signature=None):
+        """
+        Return link-time optimized PTX code for the given signature.
```

Comment:
The docstring here is a bit inconsistent with that of `inspect_asm()` - e.g. missing parameter / return descriptions, and that the code is for the device in the current context.

Reply:
Done in [e08d945](https://github.com/NVIDIA/numba-cuda/pull/326/commits/e08d9455d1297e836dd37154d13c5ac080f8c09b)